slashcommand.ts

```diff
@@ -21,7 +21,7 @@ export class JitsiSlashCommand implements ISlashCommand {
 
         const [roomName] = context.getArguments();
 
-        const url = server + (roomName ? roomName : context.getRoom().id + '-' + context.getSender().id);
+        const url = server + (roomName ? roomName : context.getRoom().id + context.getSender().id);        
```

Comment:
Please remove the extra whitespace at the end. :)

Reply:
Ok I think I have it cleaned up now (Forgot that I'm used to a very different version control system) :)